ros2bag/ros2bag/verb/play.py

```diff
@@ -24,7 +24,8 @@ class PlayVerb(VerbExtension):
 
     def add_arguments(self, parser, cli_name):  # noqa: D102
         parser.add_argument(
-            'bag_file', help='bag file to replay')
+            'input', type=str,
```

Comment:
Is this necessary?

Reply:
Artifact from previous testing. Removed in 7e7eb9b